Finjector.Web/ClientApp/src/pages/Details.tsx

```diff
@@ -68,6 +71,9 @@ const Details = () => {
   };
   const isPpmOrGlClassName =
     chartDetails?.chartType === ChartType.PPM ? "is-ppm" : "is-gl";
+  const use = () => {
+    navigate(`/teams/0/folders/0/selected/0/${chartSegmentString}`);
```

Comment:
We may not have a team folder or id for this, so I'm not sure if it is worthwhile to try and populate it.
End user shouldn't even see that if it is only from the popup

Reply:
@jSylvestre if you don't know the team and folder you can leave it out and just do `/selected/${chart...}`.  But I think it is worth checking if `chartDetails` is populated and if it is then doing the whole URL